tools/Environments/DevHome.Environments/ViewModels/ComputeSystemViewModel.cs

```diff
@@ -23,44 +23,21 @@ namespace DevHome.Environments.ViewModels;
 /// View model for a compute system. Each 'card' in the UI represents a compute system.
 /// Contains an instance of the compute system object as well.
 /// </summary>
-public partial class ComputeSystemViewModel : ObservableObject
+public partial class ComputeSystemViewModel : ComputeSystemCardBase

```

Comment:
Why a new base? Why not just use the existing ViewModel?

Reply:
The view model would become massive as we support more types of view models on the environments page. So it was better to separate them and use a template selector to switch between the view models. This allows us to have only one list instead of multiple lists for each type of view model.